Documenting policy-related CLI sub-commands (#9536)
circleci/circleci-docs: docs/guides/modules/toolkit/pages/how-to-use-the-circleci-local-cli.adoc

- `<org-id>` is the ID of your organization that owns the policy. You can find it from the CircleCI web app under menu:Organization Settings[Overview].

Note that you can filter results using any combination of the following flags:

- `--after`: Filter results to only include logs after a specific date and time.
+
Example: `--after 2025/03/14`
- `--before`: Filter results to only include logs before a specific date and time.
+
Example: `--before 2025/05/22`
- `--branch`: Filter results to only include logs for a specific branch.
+
Example: `--branch feature/fixes`

For full details on the `logs` sub-command, refer to the link:https://circleci-public.github.io/circleci-cli/circleci_policy_logs.html[CLI docs].

=== Push a policy bundle

Use the `push` sub-command to push a policy bundle.

[,shell]
----
circleci policy push <policy-dir-path> [flags]
----

- `<policy-dir-path>` is the path to the directory containing the policy files.

For full details on the `push` sub-command, refer to the link:https://circleci-public.github.io/circleci-cli/circleci_policy_push.html[CLI docs].

> === Manage policy decision settings

Author: It should actually be called "Enabling/Disabling policies" because that's all the sub-command is about. But I'm copying the text from the CLI docs 🤷‍♂️ . Maybe there will be more to it at some point.

> === Manage policy decision settings

Reviewer: ok great, I think in that case we should add the <true>/<false> options and mention the enable flag below to say that is to enable/disable the policies feature. and maybe make it clear that is all this can do? We can link here: https://circleci.com/docs/guides/config-policies/create-and-manage-config-policies/#config-policy-management-enablement

| `--branch`
| Only include logs for a specific branch.
| `--branch feature/fixes`
|===

For full details on the `logs` sub-command, refer to the link:https://circleci-public.github.io/circleci-cli/circleci_policy_logs.html[CLI docs].

=== Push a policy bundle

Use the `push` sub-command to push a policy bundle.

[,shell]
----
circleci policy push <policy-dir-path> [flags]
----

- `<policy-dir-path>` is the path to the directory containing the policy files.

For full details on the `push` sub-command, refer to the link:https://circleci-public.github.io/circleci-cli/circleci_policy_push.html[CLI docs].

=== Manage policy decision settings

Use the `settings` sub-command to get or set policy decision settings.

To read policy decision settings, run the command without any settings flags:

[,shell]
----
circleci policy settings --owner-id <org-id>
----

To set policy decision settings, run the command with the appropriate settings flags:

[,shell]
----
circleci policy settings --owner-id <org-id> --enable <true|false>
----

- `<org-id>` is the ID of your organization that owns the policy. You can find it from the CircleCI web app under menu:Organization Settings[Overview].

> - `--enable`is the flag to either enable or disable a policy evaluation. Set it to `true`to enable, `false`to disable.

Reviewer: ```suggestion
- `--enable` is the flag to either enable or disable a policy evaluation. Set to `true` to enable, `false` to disable.
```

> At the moment, the `settings` sub-command can only be used to read the settings and to xref:config-policies:create-and-manage-config-policies.adoc#config-policy-management-enablement[enable/disable a policy evaluation].

Reviewer: ```suggestion
The `settings` sub-command can only be used to read the settings and to xref:config-policies:create-and-manage-config-policies.adoc#config-policy-management-enablement[enable/disable policy evaluation for the organization].
```
